client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -521,13 +511,35 @@ const LoggedInForm = React.createClass( {
 	}
 } );
 
-const JetpackConnectAuthorizeForm = React.createClass( {
-	displayName: 'JetpackConnectAuthorizeForm',
-	mixins: [ observe( 'userModule' ) ],
+class JetpackConnectAuthorizeForm extends Component {
+	static propTypes = {
+		authAttempts: PropTypes.number,
+		authorize: PropTypes.func,
+		calypsoStartedConnection: PropTypes.bool,
+		createAccount: PropTypes.func,
+		goBackToWpAdmin: PropTypes.func,
+		goToXmlrpcErrorFallbackUrl: PropTypes.func,
+		isAlreadyOnSitesList: PropTypes.bool,
+		isFetchingSites: PropTypes.bool,
+		jetpackConnectAuthorize: PropTypes.shape( {
+			queryObject: PropTypes.shape( {
```

Comment:
I'm just sticking things here I can observe being used by this component. I've been thinking lately, in the long term, it would be _really nice_ to be able to import some standard Calypso `PropTypes` like `site`, `user`, `queryObject`, etc. It would make validation a lot more rubust and less tedious.